<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsTextMessageStyle</class>
 <widget class="QFrame" name="CSettingsTextMessageStyle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>26</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="hl_SettingsTextMessageStyle">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <spacer name="hs_TextMessageStyle">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="pb_Reset">
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_Style">
     <property name="text">
      <string>style</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pb_Font">
     <property name="text">
      <string>font</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
